migrations/882-coll-colors.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `mkt_feed_app`
```

Comment:
But background_color was null, so, I guess this is probably fine. I don't remember the logic for that.